controlplane/test/check-federated-graph.test.ts

```diff
@@ -18,7 +18,7 @@ describe('CheckFederatedGraph', (ctx) => {
 
   test('Should be able to create a federated graph, subgraphs, publish the schema and then check the graph for composition errors', async (testContext) => {
```

Comment:
Here too. Add a test that bootstrap multiple functional graph in multiple namespaces and then test the operation.

Reply:
We have a couple tests in namespace already. not needed here